pontoon/base/static/js/translate.js

```diff
@@ -271,6 +282,24 @@ var Pontoon = (function (my) {
 
 
     /*
+     * Open batch editor
+     */
+    openBatchEditor: function () {
+      var selectedCount = this.selectedEntities.length;
+
+      $('#sidebar').addClass('batch');
+
+      $('#sidebar .batch-bar')
+        .find('.variant').toggleClass('plural', selectedCount > 1).end()
+        .find('.selected-count').html(selectedCount);
+
+      $('#batch')
+        .find('button').removeClass('loading loaded')
+          .find('.message').html('');
```

Comment:
Actually, if you look at the first block, there's `.end()` at the end, which means we jump a level higher. So `.variant` and `.selected-count` are siblings.

In the 2nd block, `.message` is a child of the `button`, hence the indent.